tests/utils/tools.php

```diff
@@ -632,31 +647,12 @@ function configureFailPoint(MongoDB\Driver\Manager $manager, $failPoint, $mode,
 
     $cmd = new MongoDB\Driver\Command($doc);
     $result = $manager->executeCommand("admin", $cmd);
```

Comment:
There's no need to assign `$result` anymore if we're not going to inspect the `ok` field. Granted, I'm not even sure the old code would have ever hit the exception, since `executeCommand()` may throw an exception on its own.

Side note: since `configureFailPoint()` is now a void function, let's also remove the `return` statement in `failMaxTimeMS()` and anything else that calls this function (if there even is anything else). That will be consistent with what you did in `failGetMore()`.

Reply:
Updated